EpiserverRedirects.Tests/Tests/MapperTests.cs

```diff
@@ -31,6 +36,25 @@ public void Given_RedirectRuleDTO_Map_ReturnsRedirectRule()
             Assert.Equal(redirectRuleDto.Notes, redirectRule.Notes);
 
             Assert.Equal(redirectRuleDto.Priority, redirectRule.Priority);
+            
+            Assert.Equal(redirectRuleDto.HostId, redirectRule.HostId);
+        }
+
+        [Fact]
+        public void Given_RedirectRuleDTO_Host_Map_ReturnsRedirectRule()
```

Comment:
It's important that test name says something about what is actually tested. Here an actual test is if HostId is mapped (BTW maybe it should check HostName as well?) so maybe Given_RedirectRuleDTO_Map_ReturnsRedirectRuleWithValidHostIdAndName ?